virt/vm_networking/virt-exposing-vm-with-service.adoc

```diff
@@ -10,18 +10,22 @@ You can expose a virtual machine within the cluster or outside the cluster by cr
 
 include::modules/virt-about-services.adoc[leveloffset=+1]
```

Comment:
This `virt-about-services.adoc` module has a note in it that says: "For on-premise clusters, you can configure a load-balancing service by deploying the MetalLB Operator." Should that be hidden as well?